src/custom/constants/chainInfo/chainInfoMod.ts

```diff
@@ -90,17 +90,17 @@ export const CHAIN_INFO: ChainInfoMap = {
     label: 'Kovan',
     logoUrl: ethereumLogoUrl,
     nativeCurrency: { name: 'Kovan Ether', symbol: 'kovETH', decimals: 18 },
-  },
+  }, */
   [SupportedChainId.GOERLI]: {
     networkType: NetworkType.L1,
-    docs: 'https://docs.uniswap.org/',
+    docs: 'https://docs.cow.fi/',
     explorer: 'https://goerli.etherscan.io/',
-    infoLink: 'https://info.uniswap.org/#/',
+    infoLink: 'https://cow.fi/',
     label: 'Görli',
-    logoUrl: ethereumLogoUrl,
+    logoUrl: RinkebyLogo, // TODO: should we change this
```

Comment:
Nah, let's keep the same.
Searched around a bit and it looked like there's no icon specific for Goerli.
Unless we might want to have a different color scheme? 🤷

Reply:
Ok, I will keep Rinkeby logo for now then and we can change this if we want in the future